CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_stripe_source v0.13.0
+[PR [#87](https://github.com/fivetran/dbt_stripe_source/pull/87)] includes the following updates:
+
+## Breaking Change
+- This package assumes that amount-based fields, which as raw values are represented in the smallest denomination in Stripe, are cent-based. This PR shifts the existing conversion from cents to dollars to further upstream. Previously, currency-related fields were converted in downstream models, but now have been converted directly in staging models. Since currency-related fields now have different values, this is a breaking change. 
+
+To disable this default conversion, refer to the [README]((https://github.com/fivetran/dbt_stripe_source?tab=readme-ov-file#disabling-cent-to-dollar-conversion)) on disabling the `stripe__amount_divide` variable.
```

Comment:
We should add a note here to detail why a customer may want to disable this functionality. Also, we should also link to [this feature request](https://github.com/fivetran/dbt_stripe/issues/102) if customers find this is not enough support for them

Reply:
added!